src/Grpc.Net.Client/Balancer/Internal/SocketConnectivitySubchannelTransport.cs

```diff
@@ -49,15 +49,17 @@ namespace Grpc.Net.Client.Balancer.Internal
     /// </summary>
     internal class SocketConnectivitySubchannelTransport : ISubchannelTransport, IDisposable
     {
+        internal record struct ActiveStream(BalancerAddress Address, Socket Socket, Stream? Stream);
```

Comment:
Nice use of record structs here!

Reply:
Friendship ended with tuples.
Now record structs is my best friend.